config.json

```diff
@@ -329,6 +329,11 @@
       "slug": "kindergarten-garden",
       "difficulty": 1,
       "topics": []
+    },
+    {
+      "slug": "collatz-conjecture",
+      "difficulty": 1,
```

Comment:
Just a question not specifically related to this exercise: should we be estimating the difficulty of exercises when adding them? Currently we create a seperate issue for estimating which often takes a long time to be resolved. The person who implemented the exercise would probably be able to make a pretty good estimate for where the exercise should be, and it can always be moved if people disagree later. What do you think?